### PR TITLE
Copy current line if no selection

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @amcasey, @kevinh-ms, @ManishJayaswal

File: src/InteractiveWindow/Editor/InteractiveWindow.cs [hunk 1]
```diff
@@ -746,7 +746,26 @@ private void CutOrDelete(IEnumerable<SnapshotSpan> projectionSpans, bool isCut)
...
+        }
+
+        private static NormalizedSnapshotSpanCollection GetSelectionSpans(ITextView textView)
+        {
+            var selection = textView.Selection;
+            if (selection.IsEmpty)
+            {
+                // Use the current line as the selection.
+                var snapshotLine = textView.Caret.Position.VirtualBufferPosition.Position.GetContainingLine();
+                var span = new SnapshotSpan(snapshotLine.Start, snapshotLine.Length);
```
Comment (amcasey) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: Does this include the linebreak?  Should it?

**Comment by KevinH-MS** [comment]
LGTM :+1:

File: src/InteractiveWindow/Editor/InteractiveWindow.cs (hunk 1 above)
Comment (cston) [inline src/InteractiveWindow/Editor/InteractiveWindow.cs]: It does not include the linebreak but it should.

**Pull request merged**